File: app/src/main/java/com/itwiggle/randomchime/BootReceiver.java
package com.itwiggle.randomchime;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootReceiver extends BroadcastReceiver {
    @Override public void onReceive(Context c, Intent i) { if(Store.p(c).getBoolean("armed",false)) Scheduler.scheduleDay(c); }
}

File: app/src/main/java/com/itwiggle/randomchime/BootReceiver.kt
package com.itwiggle.randomchime

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootReceiver:BroadcastReceiver(){override fun onReceive(context:Context,intent:Intent){if(AppPrefs(context).settings().armed)AlarmScheduler.scheduleNext(context)}}
